### PR TITLE
trivial: Add quirk documentation

### PR DESCRIPTION
**Pull request by superm1**
It's a bit of manual work transcribing these over, so before doing it all what do you think of this approach?

### CODE REVIEW

**Comment by hughsie** [comment]
Like it.

**Comment by superm1** [comment]
Alright adjusted the rest of 'em.  I found a bunch of typos or bad copy/paste errors while I was transferring over, but there might be others still.

File: src/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+| Plugin            | the device ID, e.g. `DeviceInstanceId=USB\VID_0763&PID_2806`          | the plugin name, e.g. `csr`                                   | Sets the plugin to use for a specific hardware device.                                                                                                                                                                                                                                                                                                                            | n/a                                                                         | n/a           | 1.1.0                 |
```
Comment (hughsie) [inline src/README.md]: I'm not overwhelmed with the table. Would this be easier to consume if each thing was a level 3 header?

Comment (superm1) [reply src/README.md]: Well what about just collapsing the two columns not really used? Still intimidating?

Comment (hughsie) [reply src/README.md]: I think it's just too much data for the table; you have to scroll in both directions:
screenshot from 2018-11-11 08-42-14

**Comment by superm1** [comment]
Adjusted to L3 headings

File: src/README.md [hunk 2]
```diff
@@ -8,26 +8,100 @@ and then mapping out values to keys.
...
+* Key: the device ID, e.g. `DeviceInstanceId=USB\VID_0763&PID_2806`
+* Value: the device name, e.g. `ColorHug`
+* Minimum fwupd version: **1.0.2**
+### Guid
+Adds an extra GUID for a specific hardware device. If the value provided is not already a suitable GUID, it will be converted to one
+* Key: the device ID, e.g. `DeviceInstanceId=USB\VID_0763&PID_2806`
+* Value: the GUID, e.g. `537f7800-8529-5656-b2fa-b0901fe91696`
+* Minimum fwupd version: **1.0.3**
+### CounterpartGuid
+Adds an counterpart GUID for a specific hardware device. If the value provided is not already a suitable GUID, it will be converted to one.   A counterpart GUID is typically the GUID of the same device in bootloader or runtime mode, if they have a different device PCI or USB ID. Adding this type of GUID does not cause a "cascade" by matching using the quirk database.
```
[the remark this answers is not in the document]
Comment (superm1) [reply src/README.md]: adjusted

**Pull request merged**